docs/enterprise-guide/authenticating-with-saml.md

```diff
@@ -94,93 +77,102 @@ contain attributes for each User's first name, last name, and email. The asserti
+After that, type in the name of the user attribute you added in your SAML provider. In this case, we told Okta that the `metabaseGroups` attribute should be named `MetabaseGroupName`, so that's what we'll enter in the Group Attribute Name field in Metabase.
 
-After that, add the name of the user attribute you added in your SAML provider. In this case, we told Okta that the `metabaseGroups` attribute should be named `MetabaseGroupName`, so that's what we'll enter in the Group Attribute Name field in Metabase.
+![Group schema](images/saml-group-schema.png)
 
 ### Troubleshooting Tips
 
 Here are a few things to double check if you're experiencing issues setting up your SAML connection:
 
-* Verify that the application you created with your SAML provider supports SAML - sometimes other options are presented during the app creation process.
-* Read all field labels and tooltips carefully - since SAML providers each use different labeling for their fields, it's important to make sure the correct information is being placed into the correct fields.
-* Set your attributes and check your assertions! Many SAML providers make this pretty easy to do - just look for a button marked "Preview the SAML assertion."
-* Verify that the Single Sign On URL (or equivalent) that you enter on your SAML provider's website has "/auth/sso" appended to it. For instance, if you want your users to end up at ``https://metabase.mycompany.com``, the full url should be ``https://metabase.mycompany.com/auth/sso``
+- Verify that the application you created in your IdP supports SAML. Sometimes other options are presented during the app creation process.
+- Read all field labels and tooltips carefully. Since SAML providers each use different labeling for their fields, it's important to make sure the correct information is being placed into the correct fields.
+- Set your attributes and check your assertions! Many SAML providers make this pretty easy to do - just look for a button marked "Preview the SAML assertion."
+- Verify that the Single Sign On URL (or equivalent) that you enter on your SAML provider's website has "/auth/sso" appended to it. For instance, if you want your users to end up at `https://metabase.mycompany.com`, the full url should be `https://metabase.mycompany.com/auth/sso`
 
-### Disabling Password Log-in
+### Disabling password log-in
 
-Once you have configured your SAML authentication, you can choose to disable the option for users to log in via username and password. To do this, return to the main Authentication settings page and scroll to the bottom. A toggle will now be visible allowing you to make this change.
+Once you have configured SAML authentication, you can choose to disable the option for users to log in via email and password. To do this, return to the main Authentication settings page and scroll to the bottom. A toggle will now be visible allowing you to disable email+password log-in.
```

Comment:
"Email+password" seems a bit overly informal (the "+" specifically). Why not refer to it like it is on the page ("password authentication")?